pkg/reader/ccip.go

```diff
@@ -12,6 +12,10 @@ import (
 	"golang.org/x/exp/maps"
 	"golang.org/x/sync/errgroup"
 
+	"github.com/smartcontractkit/chainlink-ccip/internal/plugincommon"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/plugintypes"
+
 	types2 "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
```

Comment:
I only see this kind of type aliasing in intellij 😛 can we name this `ocr3types` instead?